apps/mailinglists/signals.py

```diff
@@ -0,0 +1,14 @@
+from django.db.models.signals import post_init
+from django.dispatch import receiver
+
+from .models import Mail, Mailinglist
+
+
+@receiver(signal=post_init, sender=Mailinglist)
```

Comment:
Do we really want this to be `post_init`?

Won't that mean we will create a `Mail` instance in our db when a MailingList object in python is created? Before is it saved to the database, which means it is potentially invalid or that it may never be saved?

I would think we could use a `post_save` here, and also make sure it is not created again for each time the MailingList model is instantiated/saved, which it looks like this will.

For that we would have to differentiate on when an instance is created and when it is updated, which we'll get from the `created` kwarg.
When updating the existing one we'll have to make sure we still the the unique one even if some or _all_ of the fields are updated